.gitignore

```diff
@@ -35,8 +35,9 @@
 # Ignore downloaded temporary files, such as the trusted root CA certificates.
 /downloads/
 
-# Ignore generated binary example files
+# Ignore generated binary files.
 **/examples/**/bin
```

Comment:
Can we just ignore `**/bin/`, like we ignore `**/target/` below?

Reply:
Done